### PR TITLE
Support for lists of modules

### PR DESCRIPTION
**Pull request by seglo**

### CODE REVIEW

**Review by ignasi35 — APPROVED** [review]
nice!

File: src/sbt-test/bom/with-modules-per-scala/build.sbt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+)
+
+lazy val billOfMaterials = project
+  .enablePlugins(BillOfMaterialsPlugin)
+  .settings(
+    name := "framework-sample-bom",
+    // Just one BOM including all cross Scala versions
+    crossVersion := CrossVersion.disabled,
+    // Create BOM in the 2.13 run
+    crossScalaVersions := crossScalaVersions.value,
```
Comment (ennru) [inline src/sbt-test/bom/with-modules-per-scala/build.sbt]: The comment and the setting don't fit together. It will create the pom.xml twice AFAICS.

**Review by ennru — APPROVED** [review]
A short note in the readme might be nice.
LGTM.

File: src/sbt-test/bom/with-modules-per-scala/build.sbt (hunk 1 above)
Comment (seglo) [reply src/sbt-test/bom/with-modules-per-scala/build.sbt]: :+1: Updated comment to reflect it's created per scala version

**Pull request merged**